pypeit/par/pypeitpar.py

```diff
@@ -542,6 +544,12 @@ def __init__(self, method=None, pixelflat_file=None, spec_samp_fine=None,
                              'filter in pixels used to determine the slit illumination function, and thus sets the ' \
                              'minimum scale on which the illumination function will have features.'
 
```

Comment:
Make this a tuple and take min and max wave with None being a value used for no tweaks or something like that.

Reply:
Agreed.

Reply:
added a new parameter.  Don't want to figure out how to 
parse `(None,9000.)` in our parser.